src/sentry/conf/server.py

```diff
@@ -1123,6 +1125,11 @@ def SOCIAL_AUTH_DEFAULT_USERNAME() -> str:
         # Run every 20 minutes
         "schedule": crontab(minute="*/20"),
     },
+    # TODO: pick a reasonable schedule to run statistical detection
+    # "detect-regressed-functions": {
+    #     "task": "sentry.tasks.statistical_detectors.run_detection",
+    #     "schedule": crontab(),
```

Comment:
Is the crontab registered at startup time? Can this be controlled using an option at runtime?

Reply:
crontabs are registered during the cron worker startup. Intervals cannot be changed once they are defined. However, celery does allow you to implement the [scheduler protocol](https://docs.celeryq.dev/en/stable/reference/celery.schedules.html#celery.schedules.schedule) and do custom stuff with it.

Reply:
I'd recommend just putting the option/feature flag inside `run_detection` and crontrol whether it runs that way, should be pretty simple

Reply:
@wedamija do you mean just pick a schedule here (like every hour), then inside `run_detection` we use a flag to decide if it should run each time?

Reply:
Sorry just saw your thread in slack, I thought you were trying to just control whether it ran at all, not have a custom schedule based on options or w/e

Reply:
doesnt seem to be worth the extra work as we're just looking to be able adjust it during testing. will go with @wedamija's suggestion and just run it hourly with an option to control if it runs and update the schedule if we want to lengthen/shorten the interval

Reply:
Avoiding a custom scheduler is the simpler option :+1: